src/test/java/uk/gov/hmcts/reform/sscs/ccd/validation/address/NonUniversalCreditAddressCanContainSpecialCharactersTest.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.sscs.ccd.validation.address;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Valid;
+import org.junit.Assert;
+import org.junit.Test;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Address;
+import uk.gov.hmcts.reform.sscs.ccd.validation.ValidatorTestBase;
+
+public class NonUniversalCreditAddressCanContainSpecialCharactersTest extends ValidatorTestBase {
```

Comment:
Might want to add a comment here to explain why UC has different address requirements to other benefit types otherwise might confuse anyone looking at this in future

Reply:
I'll add a comment to explain why this is the case

Reply:
Comments now added